app/views/providers/interrupt/blocks/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="interruption-panel">
+  <%= page_template(page_title: t(".title_html"), column_width: "full", template: :basic) do %>
+    <%= page_heading(size: "l") %>
+
+    <div class="maximize-text-width">
+      <p class="govuk-body"><%= t(".body_text") %></p>
+      <p class="govuk-body"><%= t(".options") %></p>
+      <ul class="govuk-list govuk-list--bullet">
+        <% t(".option_bullets").each do |bullet| %>
+          <li><%= bullet %></li>
+        <% end %>
+      </ul>
+    </div>
+
+    <div class='govuk-button-group'>
+      <%= govuk_button_link_to t(".applications"), providers_legal_aid_applications_path, class: "white-button", role: "button" %>
```

Comment:
Is the `white-button` class just styling the button to look like [this inverse button that's in the govuk components](https://govuk-components.netlify.app/helpers/button/#inverse-buttons)? Non-blocking, but we could consider removing the custom css at some point

Reply:
Yes, and no, we have some additional styling (bolder text, drop shadows etc) but the styling will affect multiple pages that use it.
We could add a ticket to remove the `white-button` class and replace all uses with inverse

Reply:
Ticket created